src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameReaderTests.cs

```diff
@@ -27,7 +27,14 @@ public void TestSignaturesV2_3_X()
             DataFrameReader dfr = _spark.Read();
 
             Assert.IsType<DataFrameReader>(dfr.Format("json"));
-
+            
+            Assert.Throws<NotSupportedException>(() => Assert.IsType<DataFrameReader>(
```

Comment:
Actually, the goal of #246 is to implement it such that this exception is not thrown.

Reply:
Okay, I see. I will change it accordingly. Thanks!